testnet/021eca80a25aa422d490252b053704a6d02585e401b4e85e21eb80a94728e15ab2.json

```
@@ -0,0 +1,9 @@
+{
+  "name": "DTEAM",
```

The `id` field is missing.

```suggestion
  "id": 148,
  "name": "DTEAM",
```